styles.css

```diff
@@ -93,7 +101,338 @@ ul.navbar-items {
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.first-btn button {
+  color: #efe4e4;
+  background-color: #f55800;
+  position: relative;
+  top: 350px;
+  width: 100%;
+  height: 51px;
+  border: none;
+  font-family: 'Crete Round', sans-serif !important;
+  font-style: normal !important;
+  font-weight: 700px !important;
+  font-size: 17px !important;
+  line-height: 30px !important;
+  letter-spacing: 0.03em;
+}
+
+hr {
+  position: relative;
+  bottom: 30px;
+  width: 65%;
+  border: 1px solid  #efe4e4;
+}
+
+.hr {
+  margin-left: 200px;
+}
+
+/* About me section styling */
+
+#about-page-container {
+  background-color: #1c1a19;
+  margin-top: 100px;
+}
+
+.about-me-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 100px;
+}
+
+.about-me-info {
+  flex: 1;
+  margin-right: 50px;
+  margin-left: 200px;
+  margin-top: 200px;
+  margin-bottom: 200px;
+}
+
+.about-me-image {
+  flex: 1;
+  text-align: right;
+}
+
+.about-me-image img {
+  width: 100%;
+  height: 60%;
+}
+
+.about-me-info h2 {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px !important;
+  line-height: 88px;
+  margin-bottom: 0 !important;
+}
+
+.about-me-info p {
+  color: #f4f5f7;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-top: 0 !important;
+}
+
+.about-me-info button {
+  color: #efe4e4;
+  background-color: #f55800;
+  width: 40%;
+  height: 51px;
+  border: none;
+  font-family: 'Crete Round', sans-serif !important;
+  font-style: normal !important;
+  font-weight: 700px !important;
+  font-size: 14px !important;
+  line-height: 20px !important;
+  letter-spacing: 0.03em;
+}
+
+.about-flex-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 20px;
+  margin-top: 30px;
+}
+
+.about-flex-item {
+  background-color: #1c1a19;
+  padding: 20px;
+  text-align: left;
+}
+
+.about-flex-item h3 {
+  color: #f55800;
+  font-size: 20px;
+  margin-bottom: 10px;
+  position: relative;
+  padding-left: 25px;
+}
+
+.about-flex-item:nth-child(1) h3::before {
+  background-color: #f55800;
+}
+
+.about-flex-item:nth-child(2) h3::before {
+  background-color: #f55800;
+}
+
+.about-flex-item:nth-child(3) h3::before {
+  background-color: #f55800;
+}
+
+.about-flex-item ul li {
+  margin-bottom: 15px;
+  list-style: none;
+  color: white;
+}
+
+.about-btn:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.first-desc button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.grid-btn:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.first-btn :hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+@media only screen and (max-width: 385px) {
```

Comment:
- **[OPTIONAL]** We are not working on the desktop version yet, so we don't need media queries yet. In the next milestone when we use media queries it will be difficult to create a PR that allows the code reviewer to check out your changes, I recommend removing this section to add in the corresponding milestone, remember that you have to create a new branch for each milestone.